### ISSUE TITLE
Error that prevents selection filtering on a table grid chart with a global filter

### ISSUE THREAD

**Issue by AnnieHwang** [metatron-app/metatron-discovery#217]
**Describe the bug**
글로벌 필터로 있는 컬럼으로
텍스트그리드 차트에서 선택 필터링으로도 적용하면 선택 필터가 안되는 현상

**To Reproduce**
1. 동일한 차원값, 측정값으로 하나는 바차트를, 하나는 텍스트 그리드를 만든다
2. 위의 차원값으로 글로벌 필터를 만든다
3. 대시보드를 저장한 후
4. 대시보드 열람모드에서 텍스트 그리드에서 하나의 값을 선택 필터링을 한다.

**Screenshots**
![2018-09-14_13-34-05](https://user-images.githubusercontent.com/6300003/45530095-e333f780-b823-11e8-9bd5-fc62c01fc69a.gif)

**Additional context**
글러벌 필터로 적용안 한 경우 아래와 같이 정상 동작합니다.
![2018-09-14_13-35-50](https://user-images.githubusercontent.com/6300003/45530083-d1525480-b823-11e8-9d20-85e3de1aca9d.gif)

**Comment by minjung-cho** [comment]
@AnnieHwang 글로벌 필터와 셀렉션 필터를 Merge하고 있어서 발생하는 현상으로 확인했습니다. 
개선방안으로 글로벌과 셀렉션을 AND로 거는 것이 어떨지요? 간략한 개선안 첨부합니다. 검토 부탁드립니다!
[글로벌&셀렉션필터.pptx](https://github.com/metatron-app/metatron-discovery/files/2390926/default.pptx)

**Comment by AnnieHwang** [comment]
@minjung-cho 네.. 원래 저렇게 되는게 맞는거 같아요. @kyungtaak 해당 내용을 개발일정에 잡으면 좋을것 같아요. 지금은 좀 버그 같아서요.
